web/src/pages/application/components/ApplicationForm.js

```diff
@@ -92,6 +92,9 @@ const ApplicationForm = ({ onCreate, data = {} }) => {
       >
         <Input />
       </Form.Item>
+      <Form.Item label="Token">
+        <GetApplication />
+      </Form.Item>
```

Comment:
Once the application is created then only a token can be generated. So please separate the Application form and tokens table.